pipelines/rj_ssm__compstat/task.py

```
@@ -0,0 +1,99 @@
+from typing import Any, Dict
+
+import pandas as pd
+import requests
+from prefect import task
+
+
+def requests_get_compstat(
+                        result: str,
+                        endpoint : str,
+                        url_base : str = "https://compstat.bugarintec.com.br/compstat/api/",
+                        token_estatic : str = "compstat-rio-static-token-2026-fm-qmd") -> Dict[str, Any]:
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to function default: "token_estatic"

(S107)